sections/Titans stronghold.tex

```diff
@@ -0,0 +1,174 @@
+		\item Another player (you choose) gains 1 valuable
+	\end{itemize}
+	Then flag the VII with faction cube.
+	(In solo play : no bonus!)
+	\item Ignore all yellow line in tiles I.
+	\item You can use your build token to give your resources to another player.
+	\item Two player can use your build token for exchange artifacts and/or spells.
+	\item Whenever a player visits an Obelisk, that player rolls one tresure dice and one resource dice, chooses one of the dice and resolves its outcome dice.
+	\item When all VII (not includ the titans' Stronghold) are flagged, randomly draw and shuffle the following (next page) number of neutral Unit cards from each of their corresponding decks to create a separate deck of neutral Units for the titans' Stronghold (The deck of titans' Stronghold is sometimes cut in two because otherwise certain scenarios are impossible to implement with the cards of certain extensions.)
+	\item Any time a Hero enters the titans' Stronghold they draw 5 cards from the Titans' Stronghold deck instead of drawing them from the Neutral Unit card decks. The units are placed on the Combat board (see page 29, “Neutral Unit Setup” in the Core Rulebook). Players attempt to beat the units they find at the Titans' Stronghold. Any Neutral Units defeated during Combat in the Titans' Stronghold are returned to their respective Neutral Unit decks instead of Titans' Stronghold deck. Any Neutral Units surviving Combat at the Titans' Stronghold are shuffled back into the Titans' Stronghold deck. If there are not enough Unit cards in this deck, draw as many Unit cards as there are and place them on the Combat board.
+	\item Combat in the Titans' Stronghold now costs 1 MP to extend per Combat round, just like Combat against non-Azure tier units.
+	\item Additionally, no player can:\linebreak
+	a) Attack other Heroes.\linebreak
+	b) Capture a Mine or Settlement that is already controlled.
+\end{itemize}
+
+
+
+\newpage
+
+\hommtable[]{27}{
+  \centering
+  \medskip
+  \textbf{Strength of Titans' Stronghold Armies}\\
+  \bigskip
+
+  \newcommand{\bronze}[0]{\svg[12]{bronze}}
+  \newcommand{\silver}[0]{\svg[12]{silver}}
+  \newcommand{\golden}[0]{\svg[12]{golden}}
+  \newcommand{\azure}[0]{\svg[12]{azure}}
+
+  \begin{tabularx}{\linewidth}{p{0.15\linewidth}XXXX} & \darkcell{Easy} & \darkcell{Normal} & \darkcell{Hard} & \darkcell{Impossible}\\
+    \darkcell[1.4]{1 player} 
+        & \lightcell[1.4]{3\bronze 2\silver 1\golden 1\azure}
+        & \lightcell[1.4]{2\bronze 2\silver 2\golden 1\azure}
+        & \lightcell[1.4]{2\bronze 2\silver 2\golden 2\azure}
+        & \lightcell[1.4]{1\bronze 2\silver 2\golden 3\azure}\\
+    \darkcell[1.4]{2 players} 
+	    & \lightcell[1.4]{5\bronze 5\silver 3\golden 1\azure}
+	    & \lightcell[1.4]{4\bronze 5\silver 3\golden 2\azure}
+	    & \lightcell[1.4]{2\bronze 5\silver 5\golden 3\azure}
+	    & \lightcell[1.4]{1\bronze 5\silver 7\golden 4\azure}\\
+    \darkcell[1.4]{3 players} 
```

Comment:
Starting 3 players, all cells need to be `1.6` to fit three lines in there 

![image](https://github.com/qwrtln/Homm3BG-mission-book/assets/1928291/e988b1ae-f989-4cff-a4d5-75ac61260ff4)

Or you can use similar pattern with brackets as in the Sentinels scenario (and describe in additional rules) to save one line

![image](https://github.com/qwrtln/Homm3BG-mission-book/assets/1928291/0e8de99e-4c0d-4c6a-938e-0d3e42219de1)